cpp/arcticdb/storage/python_bindings.cpp

```diff
@@ -34,7 +34,7 @@ std::shared_ptr<LibraryIndex> create_library_index(const std::string &environmen
     return std::make_shared<LibraryIndex>(EnvironmentName{environment_name}, mem_resolver);
 }
 
-void register_bindings(py::module& storage, py::exception<arcticdb::ArcticException>& base_exception) {
+void register_bindings(py::module& storage, py::exception<arcticdb::ArcticException>&) {
```

Comment:
Remove the unused argument completely?

Reply:
Done in [59436ce](https://github.com/man-group/ArcticDB/pull/1411/commits/59436ceca3bb044cf4e6412c2eceaebf9080880f)